spec/requests/chatops/slack_slash_command_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe "ApplicationChatopsController authentication", type: :request do
+  scenario "responds to ping with pong" do
+    slack_signing_secret = ENV["SLACK_SIGNING_SECRET"]
+    timestamp = Time.zone.now.to_i
+    request_body = "token=#{slack_signing_secret}&team_id=T02PF6RHYSY&team_domain=testdouble-hq&channel_id=C02NYBB3VPH&channel_name=some-channel&user_id=U02PRHH0XEV&user_name=cliff.pruitt&command=%2Fdoubleup&text=ping&api_app_id=A02PD0DUE03&is_enterprise_install=false&response_url=https%3A%2F%2Fhooks.slack.com%2Fcommands%2FT02PF6RHYSY%2F2823421496992%2F0WC0HfWeGJpHetxmF8yUmawo&trigger_id=2801968477828.2797229610916.883001cf5008d6d02fd58a3cf70f449e"
```

Comment:
**suggestion**: forgot to mention before. what are your thoughts on having this be a hash and calling `#to_query` on it instead of a long string?

Reply:
We can maybe address this later. `#to_query` sorts keys alphabetically while `post` does not.